.swiftlint.yml

```diff
@@ -1,4 +1,5 @@
 disabled_rules:
+  - large_tuple
```

Comment:
Might remove this since I don't believe our other repos have it, and just address it in areas where it comes up.

Reply:
Gonna pull this one out for now :+1: